### PR TITLE
SDNTB-164 Change user avatar translations

### PR DESCRIPTION
**Pull request by vladnicoara**

### CODE REVIEW

File: scripts/superdesk/upload/styles/upload.less [hunk 1]
```diff
@@ -146,7 +146,7 @@ user avatar uploader
                       visibility: hidden;
                     }
                     input:before {
-                      content: 'or Select from computer';
+                      content: attr(label);
```
Comment (gbbr) [inline scripts/superdesk/upload/styles/upload.less]: 👍

**Comment by gbbr** [comment]
What is the `.pot` file? Is it machine generated or user edited? Considering it is machine generated this PR LGTM :)

**Comment by petrjasek** [comment]
:+1:
@gbbr yes it's generated

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**